cal/cal/templates/category_detail.html

```diff
@@ -1,9 +0,0 @@
-{% extends "base.html" %}
-
-{% block content %}
-<div class="container">
-  <h1>Category: "{{ category.label }}"</h1>
-  <p>Cumulative hours: {{ category.hours }}</p>
```

Comment:
Add the category name and cumulative hours back to the details page. I know it's a minor detail but just want to make sure this is easily doable.